ui/raidboss/data/05-shb/dungeon/matoyas_relict.txt

```diff
@@ -0,0 +1,59 @@
+### Matoya's Relict
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0 "--Reset--" sync /00:0839:.*is no longer sealed/ window 100000 jump 0
```

Comment:
```suggestion
0 "--Reset--" sync / 00:0839:.*is no longer sealed/ window 100000 jump 0
```

Having a space here makes the regex match faster because ` 00:` will only appear at most once in line, but `00:` (with no space) might appear many more times.

Reply:
okay, understood!